express/blocks/hero-3d/hero-3d.js

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {
+  createTag,
+  getIconElement,
+  getLottie,
+  lazyLoadLottiePlayer,
+  readBlockConfig,
+// eslint-disable-next-line import/no-unresolved
+} from '../../scripts/scripts.js';
+
+const DEFAULT_DELAY = 1000;
+const MAX_NONCONFIG_ROWS = 4;
+const SCROLL_ANIMATION_URL = 'https://assets.website-files.com/62e1bd17785b4a21a5affda4/62e31da63e4adb171e1a2682_lf30_editor_jo11ftge.json';
```

Comment:
Can we store this in Helix and deliver it from the same host?

Reply:
any recommendation on how to upload a JSON file? the content is pretty unwieldy to recreate as a sheet and (just found out) the Slack bot doesn't support JSON